platforms-whitepaper/v1alpha1/paper.md

```diff
@@ -375,44 +373,56 @@ true measure of the success of a platform.
 
 ## Capabilities of platforms
 
-As described earlier, a platform for cloud-native computing offers and composes
+As we've described, a platform for cloud-native computing offers and composes
 capabilities and services from many supporting providers. These providers may be
 other teams within the same enterprise or third parties like cloud service
-providers. Platforms enable use of the capabilities and services from these
-providers by wrapping them with consistent web portals, documentation, code
-templates, and programmable APIs and tools. 
-
-In a nutshell, platforms bridge from underlying capability _providers_ to
-platform users like application developers; and in the process implement and
-enforce desired practices for security, performance, cost governance and
-consistent experience. The following graphic illustrates the relationships
-between product, platform, and capability providers.
+providers. In a nutshell, platforms bridge from underlying _capability
+providers_ to platform users like application developers; and in the process
+implement and enforce desired practices for security, performance, cost
+governance and consistent experience. The following graphic illustrates the
+relationships between products, platforms, and capability providers.
 
 <img src="assets/platform_components.png" width=600px />
 
-We've described the attributes and values of platforms and platform teams. Now
-let's describe the kinds of capabilities typically required in cloud-native
-computing platforms. Our goal is to guide platform engineers in ensuring their
-platform is complete.
-
-1. **Web portal** for provisioning and observing capabilities
+We've focused in this paper on how to construct a good platform and platform
+team; now in this last section we'll describe the capabilities a platform may
+actually offer. This list is intended to guide platform builders and includes
+capabilities typically required by cloud-native applications. As we've noted
+throughout though, a good platform reflects its users' needs, so ultimately
+platform teams should choose and prioritize the capabilities their platform
+offers together with its users.
+
+Capabilities may comprise several _features_, meaning aspects or attributes of
+the parent capability's domain. For example, observability may include features
+for gathering and publishing metrics, traces and logs as well as for observing
+costs and energy consumption. Consider the need and priority for each feature or
+aspect in your organization. Later CNCF publications may expand on each
+domain further.
+
+Here are capability domains to consider when building platforms for cloud-native
+computing:
+
+1. **Web portals** for observing and provisioning services
```

Comment:
Should this also be capabilities instead of services, in alignment with the next bullet?
```suggestion
1. **Web portals** for observing and provisioning capabilities
```

Reply:
Interesting to bring this up. I think portals may be for provisioning and observing _both_ applications and the capabilities they rely on, take for example Backstage's [software catalog](https://backstage.io/docs/features/software-catalog/). How about "for observing and provisioning products and capabilities" and a parallel language for APIs/CLIs?

Reply:
I like that!